guide.md

```diff
@@ -39,32 +39,30 @@ Illustration sources
 
 # The bitcoin design guide
 
-**Bitcoin is an incredibly open and public technology that we can all use, be part of, and contribute to. It is also a complex technology that takes time to understand, with a global audience and diverse uses cases. Therefore it's best to combine efforts, share knowledge and support each other in building great Bitcoin products.**
+**Bitcoin is a global payment protocol that anyone with Internet access can participate in and contribute to. However, because bitcoin’s use cases are as vast as its user base, building bitcoin applications can be complicated.**
```

Comment:
Haha, repeat of the capitalization discussion?

Reply:
I'm a big lowercase b for bitcoin, capital I for Internet guy. by all means feel free to swap it back. I think it's simpler this way and easier for others to track but really not very important to me

Reply:
We tried to standardize this here https://bitcoin.design/guide/getting-started/technology-primer/#is-it-bitcoin-or-bitcoin but the thing is it's not consistent in our guide on all places. I guess we should stick to what's defined at least.

Reply:
Addressed in https://github.com/BitcoinDesign/Guide/pull/220/commits/ae8f0c16197237509dea0bfc4dae5debec0f0d72